habitat-hitl/habitat_hitl/_internal/networking/networking_process.py

```diff
@@ -360,9 +495,9 @@ async def start_http_availability_server(
     async def http_handler(request):
         # return an HTTP code to indicate available or not
         code = (
-            networking_config.http_availability_server.code_unavailable
-            if network_mgr.has_connection()
-            else networking_config.http_availability_server.code_available
+            networking_config.http_availability_server.code_available
+            if not network_mgr.has_connection()
+            else networking_config.http_availability_server.code_unavailable
```

Comment:
This is the code that determines whether a server is available. It is used by a load balancer in single-user and matchmaker in multi-user.

We only consider the server available when there is no connection.

In the multi-user case, the matchmaker dispatches every other user to the previous user's location.